seismicpro/gather/gather.py

```diff
@@ -1273,6 +1274,9 @@ def plot(self, mode="seismogram", *, title=None, x_ticker=None, y_ticker=None, a
             * `log`: set y-axis to log scale. If `True`, formatting defined in `y_ticker` is discarded,
             * Any additional arguments for `matplotlib.pyplot.hist`.
 
+        A 2d array matching the size of the gather, a 1d array with length equal to gather.n_traces or list of these
+        arrays can be drawn on top of the seismogram or wiggle plot if passed as `mask`.
+
```

Comment:
```suggestion
        Some areas of a gather may be highlighted in color by passing optional `masks` argument.
```
And join it to the next paragraph. No need to describe available options here, you do it below.

Reply:
done